main/src/main/java/cgeo/geocaching/connector/oc/OCUserInfo.java

```diff
@@ -0,0 +1,19 @@
+package cgeo.geocaching.connector.oc;
+
+import cgeo.geocaching.connector.UserInfo;
+
+import androidx.annotation.NonNull;
+
+public class OCUserInfo extends UserInfo {
```

Comment:
No need for an OC-specific extension imho. I would recommend to extend UserInfo with an (optional) field `remainingFavoritePoints ` instead (with default -1 if user or platform has no fav points)

Reply:
I fixed that by adding additional constructor in UserInfo class